### ISSUE TITLE
Replace About and Support text in lightbox

### ISSUE THREAD

**Issue by hydrolet** [endlessm/kolibri-explore-plugin#807]
Current text:

**Endless Key — learning and discovery resources, no internet required.**
The Endless Key unlocks the potential for student curiosity and choice through independent learning. Curated to be interest-based and inclusive, our collection of content encourages students to discover knowledge and skills that will support them now and in their futures. The Endless Key is designed to complement the broadband-for-all roll-out in the United States and provide offline students with equitable access to engaging content. Students can access Endless Key content by downloading the Endless Key app (available for ChromeOS and Windows). After the initial download, no internet is required to interact with the content.

**Support**
You can find more information on how to use your Endless Key on The Quick Start Guide document. For more information, when you have an internet connection, you can access Endless Support Site at https://support.endlessos.org/en/endless-key.

Proposed text:

[Endless Key](https://www.endlessos.org/key) provides great educational content and learning resources unlocking the potential for student curiosity and choice through independent learning—whether kids are online or offline. Designed to provide a robust, engaging, and safe digital learning experience, Endless Key is available as a downloadable app for Chromebooks and Windows PCs, and **as a flatpak for Linux distributions**. Once installed and set up, Endless Key can work entirely offline.

**Support**
To find more information on how to use the Endless Key app, you can access the Endless Support Site at https://support.endlessos.org/en/endless-key.

@erikos @cassidyjames Can you please look at the proposed text above and give your feedback or suggestions? I'm not sure how to word the Linux part (in bold).

**Comment by cassidyjames** [comment]
I actually wonder… should we remove this in-page about/support dialog since we're already _also_ shipping the native GTK one? It feels a little awkward to have both imho. 😄 

![Screenshot from 2023-09-15 15-30-48](https://github.com/endlessm/kolibri-explore-plugin/assets/611168/5ebda04f-7765-4f72-ace7-527ff89b7049)

The native one can actually store a whole lot of information into a nice compact format, plus it would be familiar to users of other GTK apps on Endless OS and Linux. The one thing we'd lose if we dropped the in-page lightbox is the ability to do the cheat code, I think.

**Comment by cassidyjames** [comment]
@hydrolet hm, is it necessary to have all of this copy in the in-app About dialog? It feels like things the user would already know since they got it installed in the first place… but I may be missing something.

Regarding the copy itself, Flatpak is a technical detail to be honest (like how the Windows version is an exe and the ChromeOS version is an APK). The more interesting thing is _from where_ you get them imho, so I'd recommend just treating them equally, like:

>Endless Key is available as a downloadable app for ChromeOS on Google Play, Windows on the Microsoft Store, and Linux on Flathub.

**Comment by manuq** [comment]
@hydrolet is asking review of the text above @erikos @cassidyjames. Please let me know when this is good to replace.

**Comment by manuq** [comment]
> I actually wonder… should we remove this in-page about/support dialog since we're already _also_ shipping the native GTK one? It feels a little awkward to have both imho. 😄

We could especial-case the flatpak for this. On Android I don't see any About in the app information once the app is installed (only the description in the App store). For the online instance we'll still need the in-app About. I haven't looked at the situation for Windows.

**Comment by dylanmccall** [comment]
> > I actually wonder… should we remove this in-page about/support dialog since we're already _also_ shipping the native GTK one? It feels a little awkward to have both imho. 😄
> 
> We could especial-case the flatpak for this. On Android I don't see any About in the app information once the app is installed (only the description in the App store). For the online instance we'll still need the in-app About. I haven't looked at the situation for Windows.

I feel like the Explore plugin has two different About dialogs squashed together. One of them has release information, a description and privacy policy for Endless Key as a whole, and support links. The other has attributions for various bits of artwork, and diagnostic information about the Explore plugin itself (including _its_ version number, which doesn't have to correspond to the desktop app's version number).

The desktop app is definitely in a position to replace the first one of those. But I think the other one - at least the Credits part - still needs to live in the Explore plugin either way. And the privacy policy part worries me a bit. It would be really awkward if we had a duplicate copy of it in the Linux desktop app and otherwise it's in the Explore plugin…

**Comment by jofilizola** [comment]
@cassidyjames the user might have gotten the app pre-loaded on the Endless Laptop or a parent could find it installed by the school on the computer their kid is using - the person using it might not have installed it in the first place. So its nice to have some info, and what other platforms it can be found. Assuming this lives on the plugin and it's better to be the same on EOS, Android, Linux and the Online demo.

It has artwork credit info, which ideally should live in the channels descriptions, but those channels are ingested by scripts and it did not seem worth it ingesting them again to include that information.

On the Linux App case, could we just hide whatever is repeating in that dialog (release info, support and the about copy) and just show the privacy policy link at the footer.

By the way: I was about to open an issue: all of the links on the App Menu of the Endless Key link to Learning Equality's website. Including the "website" here:

![image](https://github.com/endlessm/kolibri-explore-plugin/assets/51095367/dff8c984-56b5-4b65-a24c-2e8b5676d18c)

**Comment by jofilizola** [comment]
- https://github.com/endlessm/endless-key-flatpak/issues/71

**Comment by cassidyjames** [comment]
I would almost just remove the native About window if most/all of this information is going to be in the explore plugin. The About dialog explicitly designed to host all of this kind of information, though, in a way that is consistent across apps on Linux and Endless OS. So at the very least we should make sure the information matches/is accurate, but if it's too complicated to get it into the About window and/or we want the experience to exactly mirror ChromeOS and Windows, then we could consider removing the About window entirely.

**Comment by manuq** [comment]
Can we go ahead with the text edits now (in current in-app About), and split the native About conversation into another ticket?

**Comment by jofilizola** [comment]
Good idea, @manuq I've updated the description with the changes, so we can go ahead with that!
Would you create the new task to track the native about dialog?
